code-scanning/crda.yml

```diff
@@ -0,0 +1,132 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+# This workflow performs a static analysis of your source code using
+# Red Hat CodeReady Dependency Analytics.
+
+# Scans are triggered:
+# 1. On every push to default and protected branches
+# 2. On every Pull Request targeting the default branch
+# 3. On a weekly schedule
+# 4. Manually, on demand, via the "workflow_dispatch" event
+
+# 💁 The CRDA Starter workflow will:
+# - Checkout your repository
+# - Setup the required tool stack
+# - Install the CRDA command line tool
+# - Auto detect the manifest file and install the project's dependencies
+# - Perform the security scan using CRDA
+# - Upload the SARIF result to the GitHub Code Scanning which can be viewed under the security tab
```

Comment:
Is this performed by the crda action? If not, then it can be added as a step like this :
```
 - name: Upload the SARIF result
      uses: github/codeql-action/upload-sarif@v2
      with:
        sarif_file: results.sarif
```

Reply:
@Phantsure Yes, SARIF upload is performed by this action itself, no need to use `upload-sarif` action.